File: pset1/Cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
    float plata;
    do
    {
        plata = get_float("decime el valor de tu cambio: "); //dice el valor de tu cambio
    }

    while (plata < 0);
    int centavos = round(plata * 100); //se multiplica por cien para que sea un numero entero
    int monedas = 0;
    while (centavos >= 25)
    {
        centavos -= 25; // mientras sea el valor a 25 devuelva ese valor y asi con todos
        monedas ++;
    }
    while (centavos >= 10)
    {
        centavos -= 10;
        monedas ++;
    }
    while (centavos >= 5)
    {
        centavos -= 5;
        monedas ++;
    }
    while (centavos >= 1)
    {
        centavos -= 1;
        monedas ++;
    }


    printf("%i\n", monedas); //imprime la cantidad de monedas mas optimas

    //{}


}

File: pset6/Python/cash/cash.py
from cs50 import get_float
while True:
    entrada_plata = get_float ("Decime un valor: ")
    if (entrada_plata > 0) :
        break

monedas = 0
centavos = round(entrada_plata *100)
while centavos >= 25:
    centavos -= 25
    monedas += 1
while centavos >= 10:
    centavos -= 10
    monedas += 1
while centavos >= 5:
    centavos -= 1
    monedas +=1
while centavos >= 1:
    centavos -= 1
    monedas += 1
print (f"{monedas}")